analysis/import/SourceCodeParser/src/test/kotlin/de/maibornwolff/codecharta/importer/sourcecodeparser/metrics/ProjectMetricsTest.kt

```diff
@@ -26,7 +26,7 @@ class ProjectMetricsTest {
 
         addMetricToFileInProject(projectMetrics, "foo", "mcc", 99)
 
-        Assertions.assertThat(projectMetrics.projectMetrics["foo"]).isEqualToComparingFieldByField(expected)
+        Assertions.assertThat(projectMetrics.projectMetrics["foo"]!!.fileMetrics).isEqualTo(expected.fileMetrics)
```

Comment:
Seems like we compare less things in these tests now? Could be change that again?